pilot/pkg/config/kube/gateway/testdata/http.yaml

```diff
@@ -315,3 +316,76 @@ spec:
           port: 80
       retry:
         attempts: 0
+---
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
+metadata:
+  name: http-route-cors
+  namespace: default
+spec:
+  hostnames:
+  - "cors.domain.example"
+  parentRefs:
+    - name: gateway
+      namespace: istio-system
+  rules:
+    - backendRefs:
+        - kind: Service
+          name: httpbin
+          port: 80
+      filters:
+        - cors:
+            allowCredentials: true
+            allowOrigins:
+            # - '*' # This will be allowed in the future, probably https://github.com/kubernetes-sigs/gateway-api/issues/3648#issuecomment-2735208553
+            # - '*.com'
+              - "https://example.com"
+            allowMethods:
+              - GET
+              - HEAD
+              - POST
+            allowHeaders:
+              - Accept
+              - Accept-Language
+              - Content-Language
+              - Content-Type
+              - Range
+          type: CORS
+---
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
+metadata:
+  name: multiple-inferencepool-backend-refs
```

Comment:
the diff is a little weird. I think this HTTPRoute already exists there, why does it show like it adds it?